MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs

```diff
@@ -128,5 +133,26 @@ public static bool UseUnixSocket(this IConfiguration configuration)
         /// <returns>The sqlite cache size.</returns>
         public static int? GetSqliteCacheSize(this IConfiguration configuration)
             => configuration.GetValue<int?>(SqliteCacheSizeKey);
+
+        /// <summary>
+        /// Gets whether second level cache disabled from the <see cref="IConfiguration" />.
+        /// </summary>
+        /// <param name="configuration">The configuration to read the setting from.</param>
+        /// <returns>Whether second level cache disabled.</returns>
+        public static bool GetSqliteSecondLevelCacheDisabled(this IConfiguration configuration)
+        {
+            var disableSecondLevelCacheConfig = configuration.GetValue<string?>(SqliteDisableSecondLevelCacheKey);
```

Comment:
Can't we use `GetValue<bool>(SqliteDisableSecondLevelCacheKey, false)`?

Reply:
We can, but that will make this mush less user-friendly, as the user will have to type in a valid bool string (TRUE or FALSE, case insensitive) and all other value will make dotnet complain and error out. This is more user friendly.

Reply:
This shouldn't be a user-friendly config option, if they change the config then they must conform to the accepted values.

Reply:
But not even supporting 1 and 0 which is super common for env vars is a little bit too restrictive.

Reply:
I'd agree with gnattu here, sure it's not *common* to use but making it deliberately obtuse to use is not good design. If this is needed to get sensible boolean-like flags (e.g. `true`, `TRUE`, `1`, etc.) then I think it's worthwhile.